constants/constants.go

```diff
@@ -8,21 +8,23 @@ import (
 )
 
 const (
-	KubernetesVersion                     = "v1.10.11"
+	KubernetesVersion                     = "v1.11.7"
 	CNIVersion                            = "v0.6.0"
 	BaseInstallDir                        = "/opt/bin"
 	CNIBaseDir                            = "/opt/cni/bin"
 	CNIConfigDir                          = "/etc/cni"
 	CNIStateDir                           = "/var/lib/cni"
 	SystemdDir                            = "/etc/systemd/system"
 	FlannelVersion                        = "v0.10.0"
+	MetalLBVersion                        = "master"
 	DefaultPodNetwork                     = "10.244.0.0/16"
 	DefaultDNSIP                          = "10.96.0.10"
 	DefaultServiceSubnet                  = "10.96.0.0/12"
 	DefaultDNSDomain                      = "cluster.local"
 	DefaultRouterID                       = 42
 	KubeadmConfig                         = "/tmp/kubeadm.yaml"
-	KubeDNSVersion                        = "1.14.8"
+	CoreDNSVersion                        = "1.1.3"
```

Comment:
Hi Daniel, sorry forgot to ask during code walk through, for upgrade scenarios do we need an explicit `kubectl delete` for `kubedns`

Reply:
Yes, but that will be done in cctl.